app/controllers/application_controller.rb

```diff
@@ -1,5 +1,14 @@
 class ApplicationController < ActionController::Base
+  before_filter :configure_permitted_parameters, if: :devise_controller?
+
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  protected
```

Comment:
This is probably 100% correct. I dont super understand protected myself. So if you arent 100% sure you know what this does and why it is different from `private` I would look for some documentation or blog posts on it. Or bug Jared. He super knows that stuff